raiden_installer/tokens.py

```diff
@@ -107,6 +108,21 @@ def formatted(self):
     def as_wei(self) -> Wei:
         return Wei(self.value * (10 ** self.currency.decimals))
 
+    @property
+    def as_fiat(self):
+        if not self.value:
+            return 0
+        response = requests.get(
+            'https://pro-api.coinmarketcap.com/v1/tools/price-conversion',
+            params={'symbol': self.ticker, 'amount': self.value, 'convert': 'USD'},
+            headers={
+    	        'Accept': 'application/json',
+    	        'X-CMC_PRO_API_KEY': '30eeec47-754b-4fd1-85c5-a35edf88ff5b'
```

Comment:
this should come from a constant and/or `os.environ.get("CMC_PRO_API_KEY", "")`
Question: does this work (rate limited) without an API-Key?

Reply:
Unfortunately does not work at all without the API keys
https://coinmarketcap.com/api/features